sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/json/JsonFileFormat.scala

```diff
@@ -26,7 +26,8 @@ import org.apache.hadoop.mapreduce.{Job, TaskAttemptContext}
 import org.apache.spark.internal.Logging
 import org.apache.spark.sql.{AnalysisException, SparkSession}
 import org.apache.spark.sql.catalyst.InternalRow
-import org.apache.spark.sql.catalyst.json.{JacksonGenerator, JacksonParser, JSONOptions}
+import org.apache.spark.sql.catalyst.json.{JacksonGenerator, JacksonParser}
+import org.apache.spark.sql.catalyst.json.{JSONOptions, JSONOptionsInRead}
```

Comment:
question/nit: Why is this split on 2 import statements this way?

Reply:
Just to make Scala's code style checker happy and to fit to the 100 chars restriction. I will check if the style checker is happy if I fold imports to one line.